src/components/status/StatusSelector.jsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { connect } from "react-redux";
+import { setStatus } from "../../state/status/actions";
+import PropTypes from "prop-types";
+
+export class StatusSelector extends React.Component {
+  constructor(props) {
+    super(props);
+    this.onChange = this.onChange.bind(this);
+  }
+
+  onChange(event) {
+    this.props.dispatch(setStatus(event.target.value));
+  }
+
+  render() {
+    return (
+      <div>
+        <h4>Status:</h4>
+        <select id="status" onChange={this.onChange} value={this.props.value}>
+          <option value="">All</option>
```

Comment:
one way that you may choose to deal with these statuses is to create constants somewhere that you can `export` them so you can reuse them through the app like:
```js
export const STATUS_REJECTED = "rejected";
export const STATUS_DRAFT = "draft";
export const STATUS_REVIEW = "review";
...

export const STATUSES = [
  STATUS_REJECTED,
  STATUS_DRAFT,
   ...
]
```

and then here you could do something like:
```js
import { STATUSES } from "statuses.js";

<select>
  <option>All</option>
  {STATUSES.map(status => (
    <option value={status} key={status}>
      {status.charAt(0).toUpperCase()}{status.slice(1)}
    </option>
  ))}
</select>
```

Reply:
Ooooo constants is a good idea, thanks!! :D